### DISCUSSION TITLE
Welcome to typemill Discussions!

### DISCUSSION THREAD

**Discussion by trendschau** [typemill/typemill#417] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robs-fotoalbum** [comment]
Hello,

I just finished the upgrade to V2. I had two plugins active: search and ebook. The first is running in V2, ebook 2 is not yet working. There is an error message regarding missing navigation. Is this not ready or a known error? Should I open an issue for that? 
The ebook links on the pages are missing, but they are switched on. This is maybe related to https://github.com/typemill/typemill/issues/423 .

V2 looks clean and well-structured, the update itself was not so easy, sometimes setting in the yaml file were not saved directly, sometimes multiple checkboxes could be set for one attribute (e.g. put information on top or bottom of a page), but with a mix of direct edit and via the settings I was able to manage all steps so far.

The language of the frontend is a mixture of English and German currently, I guess this is an own issue. Password recovery is not yet working either.

Looking forward to new ebook plugin working.

Cheers
Rob

**Comment by trendschau** [comment]
thank your for your feedback, I will check the ebook plugin asap. Password reset should work with next release again. The language is an ongoing optimization, I will improve step by step. Regarding the settings: How did you manage your setup? Did you delete the settings and reconfigure your system or did you keep the old settings with version 2? I think the best way is to delete the settings and to reconfigure the system, because version 2 has been rewritten in many parts and the logic is quite different from version 1.

**Reply by robs-fotoalbum** [reply]
I copied the settings.yaml from version 1 to the update. The layout is fine, but the issues might come from this approach. Maybe I could just overwrite the current settings.yaml with a fresh V2 version and edit it step by step via the web interface?

**Reply by robs-fotoalbum** [reply]
I now tried a fresh installation on a new sub-domain with the 2.0.3. zip here https://typemill.net/getting-started/installation but the setup returned "404 The requested URL was not found on this server."

**Reply by robs-fotoalbum** [reply]
...solved - there came no .htaccess file with the fresh installation. Copied one from the other installation and now I'm in the fresh 2.0.3. Will experiment with the settings.yaml's a bit a see what's happening.

**Comment by trendschau** [comment]
The htaccess is definately part of the zip-file that you can download on typemill.net and it is also part of of the github repository. With an old htaccess typemill v2 won't work correctly. 

I wrote a pretty comprehensive upgrade guide on https://typemill.net/typemill-v2/upgrade-guide and did upgrades for several installations without problems. Of course there can be problems that I did not face yet, but my advice is to follow these guides.

**Reply by robs-fotoalbum** [reply]
Yes, I read the upgrade guide. The htaccess was not visible in my finder due to settings. This I have changed, performed a fresh installation again, that is working, but the migration not. 

The content headings are visible in the frontend, but the pages cannot be found after click. The migrated content is not visible in the editor. 

Probably because I switched to a new subdomain for the new migration test installation? Where would I set the new subdomain?

**Reply by robs-fotoalbum** [reply]
New behaviour: it seems to work now, after I have depublished the welcome and cyanin theme folders. I don't know why, but it seems to work now :-)

Now I will work on the settings and see, if the tabs for content and meta will stay visible in the editor.

**Reply by trendschau** [reply]
good to hear!! :)
